src/uniqBy.js

```diff
@@ -0,0 +1,38 @@
+var _contains = require('./internal/_contains');
+var _curry2 = require('./internal/_curry2');
+
+
+/**
+ * Returns a new list containing only one copy of each element in the original list, based
+ * upon the value returned by applying the supplied function to each list element. Prefers
+ * the first item if the supplied function produces the same value on two items. `R.equals` is used for comparison.
+ *
+ * @func
+ * @memberOf R
+ * @category List
+ * @sig (a, b) -> [a] -> [a]
```

Comment:
Rather than `(a, b)` we should have `(a -> b)`, a function from type `a` to type `b`.